### ISSUE TITLE
world_void vs world_test

### ISSUE THREAD

**Issue by ForTheWout** [ForTheCrown/FTC#25]
We should split the bank, the dungeons and any other spot players can tp to from the other stuff that's currently in the void world. That way, if a player manages to break out somehow, they can't find random chests with creative items in them oops

 By doing this, we can start using actual void worlds too instead of a fake flat plains one lol

**Comment by JulieWoolie** [comment]
I would love to do this, a small problem is that there's a lot of hardcoded values that require the void world to be called `world_void` and for everything in that world to be in specific places. Most pressingly, the dungeons, which have the boss rooms hardcoded

**Comment by ForTheWout** [comment]
Yeah most of the command blocks for the bank vaults also use "hardcoded" locations in them, so it'll be a pain to do this.
Think it's worth it though, I'll start with copying stuff to a new world.

**Comment by ForTheWout** [comment]
New dungeon coords:
- [ ] Zhambie: 10.5, 33, 108.5
- [ ] Zhambie helpers:
- 5.5, 32, 114.5
- 15.5, 32, 114.5
- 5.5, 32, 102.5
- 15.5, 32, 102.5
- [ ] Make Josh (wither skeleton) drop goo
- [ ] Skalatan: 98.5 20 135.5
- [ ] Hidey Spidey: 123.5 9 235.5
- [ ] Drawned: 78.5 -21 -10.5
- [ ] Emo: -277 37 44.5
- [ ] Emo shulker locations:
- -285.5, 34, 52.5
- -269.5, 34, 52.5
- -285.5, 34, 36.5
- -269.5, 34, 36.5

**Comment by JulieWoolie** [comment]
What is there left to do for this one?

**Comment by ForTheWout** [comment]
I wanted to rename the worlds lastly, but I remember you talked about that being easier said than done?
Ideally the world currently called `world_void` should be renamed to something like `world_test`, `world_admin` or `world_playground`
